### PR TITLE
fix git url dependency

### PR DESCRIPTION
**Pull request by craigmichaelmartin**
cf https://docs.npmjs.com/files/package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "abp-filter-parser-cpp": "^1.0.22",
     "electron-localshortcut": "^0.6.0",
-    "electron-prebuilt": "https://github.com/brave/electron-prebuilt",
+    "electron-prebuilt": "git+https://github.com/brave/electron-prebuilt",
```
Comment (eliperelman) [inline package.json]: This could be shortened to:

```json
"electron-prebuilt": "brave/electron-prebuilt",
```

File: package.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
     "tracking-protection": "^1.0.7",
     "url-loader": "^0.5.7",
     "urlutil.js": "^0.1.1",
-    "vibrant": "https://github.com/jariz/vibrant.js#1.0"
+    "vibrant": "git+https://github.com/jariz/vibrant.js#1.0"
```
Comment (eliperelman) [inline package.json]: This could be shortened to:

```json
"vibrant": "jariz/vibrant.js#1.0",
```

**Comment by eliperelman** [comment]
I made some comments, which would also make the git dependencies consistent with the `repository` field in package.json.

**Comment by craigmichaelmartin** [comment]
Great, thanks @eliperelman!

**Comment by bbondy** [comment]
Looks great, thanks!

**Pull request merged**